### DISCUSSION TITLE
Using users created at image build time

### DISCUSSION THREAD

**Discussion by RafaelPalomar** [89luca89/distrobox#1493] [Q&A]
❯ podman build -t test . 
STEP 1/2: FROM debian:bookworm
STEP 2/2: RUN useradd -s /bin/bash testuser
--> Using cache 4f1e67e8433f722c4de10339ff81f167e1dc03b21f2398db3c47f31bd27aff18
COMMIT test
--> 4f1e67e8433f
Successfully tagged localhost/test:latest
4f1e67e8433f722c4de10339ff81f167e1dc03b21f2398db3c47f31bd27aff18

/tmp/test 
❯ DBX_CONTAINER_ALWAYS_PULL=0 distrobox create -i test
Creating 'test' using image test	 [ OK ]
Distrobox 'test' successfully created.
To enter, run:

distrobox enter test

/tmp/test 
❯ distrobox enter test
Starting container...                   	 [ OK ]
Installing basic packages...            	 [ OK ]
Setting up devpts mounts...             	 [ OK ]
Setting up read-only mounts...          	 [ OK ]
Setting up read-write mounts...         	 [ OK ]
Setting up host's sockets integration...	 [ OK ]
Integrating host's themes, icons, fonts...	 [ OK ]
Setting up package manager exceptions...	 [ OK ]
Setting up package manager hooks...     	 [ OK ]
Setting up dpkg exceptions...           	 [ OK ]
Setting up apt hooks...                 	 [ OK ]
Setting up distrobox profile...         	 [ OK ]
Setting up sudo...                      	 [ OK ]
Setting up user groups...               	 [ OK ]
Setting up kerberos integration...      	 [ OK ]
Setting up user's group list...         	 [ OK ]
Setting up existing user...             	 [ OK ]
Setting up user home...                 	 [ OK ]
Ensuring user's access...               	 [ OK ]

Container Setup Complete!
📦[rafael@test test]$ id testuser
id: ‘testuser’: no such user

**Comment by happymab** [comment]
Look at the error message when trying to create the distrobox: Distrobox named 'test' already exists.
Remove the distrobox test first and try again.

**Reply by RafaelPalomar** [reply]
Hi @happymab. Thanks for the reply. Sorry I guess I messed up with copying the results. I updated the snippet. The error persists. Could you try to reproduce it? Thanks!
